pubspec.yaml

```diff
@@ -80,6 +80,21 @@ dependencies:
       url: git://github.com/linagora/jmap-dart-client.git
       ref: master
 
+  # http_parser
+  http_parser: 4.0.0
+
+  # file_size
+  filesize: 2.0.1
+
+  # flutter_widget_from_html
+  flutter_widget_from_html: 0.6.2
```

Comment:
how about https://pub.dev/packages/flutter_html?

Reply:
render error

Reply:
What kind of error?

Reply:
Unable to render when tags are nested.